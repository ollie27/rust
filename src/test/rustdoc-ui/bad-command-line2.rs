// compile-flags:-Cdoes-not-exist
