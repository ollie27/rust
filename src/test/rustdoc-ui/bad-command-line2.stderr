error: unknown codegen option: `does-not-exist`

